src/components/AuthLayout/index.js

```diff
@@ -56,19 +45,20 @@ class AuthLayout extends Component {
   render() {
     let { heading } = this.props;
     const { backOpt, toPreview } = this.props;
-    const { isOpen } = this.state;
 
-    const dropdownItems = [
-      <DropdownItem key="german" className={styles.form}>
-        German
-      </DropdownItem>,
-      <DropdownItem key="hindi" className={styles.form}>
-        Hindi
-      </DropdownItem>,
-      <DropdownItem key="french" className={styles.form}>
-        French
-      </DropdownItem>,
-    ];
+    // P.S. : Will be added later with drpdown for language localization
```

Comment:
Not sure about keeping commented-out code ... (see below).

Reply:
removed and will be tracked with #115